packages/bot-skeleton/src/scratch/utils/index.js

```diff
@@ -359,7 +378,8 @@ export const runInvisibleEvents = callbackFn => {
     Blockly.Events.enable();
 };
 
-export const updateDisabledBlocks = (workspace, event) => {
+export const updateDisabledBlocks = event => {
```

Comment:
Could we revert this change? Try to keep utils pure-_ish_ by passing in the objects it modifies.

Reply:
yeah sure. I was thinking this function doesn't used in any other workspace beside main workspace.

Reply:
okayy